AppCreationScripts/apps.json

```diff
@@ -0,0 +1,33 @@
+{
+  "Sample": {
+    "Title": "Calling Microsoft Graph from an Android app",
+    "Level": 200,
+    "Client": "Android"
+  },
+  "AppRegistrations": [
+    {
+      "x-ms-id": "active-directory-android-java",
+      "x-ms-name": "android-java",
+      "x-ms-version": "1.0",
+      "replyUrlsWithType": [
+        {
+          "url": "msal{AppId}://auth",
```

Comment:
This is the old redirect url pattern. the current one would be msauth://{package_name}/{signature_hash}
i.e. msauth://com.azuresamples.msalandroidapp/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D